client/vehicle.lua

```diff
@@ -0,0 +1,730 @@
+local HasAlreadyEnteredMarker, LastZone = false, nil
```

Comment:
This also needs to be put on a new client lua file.

Reply:
client/vehicle.lua was removed po sa commit #3